src/containers/app/CreateApp.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FormControl, FormGroup, ControlLabel, Button, Alert } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import AsyncContent, { AsyncStatePropType } from '../../components/asynccontent/AsyncContent';
+import { createAppRequest } from './AppActionFactory';
+
+
+class CreateApp extends React.Component {
+
+  static propTypes = {
+    actions: PropTypes.shape({
+      onCreateApp: PropTypes.func.isRequired
+    }).isRequired,
+    createAppAsyncState: AsyncStatePropType.isRequired
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      description: '',
+      name: '',
+      appTypeIds: [],
```

Comment:
Just a thought... theoretically, all AppType ids must exist before creating a new App, so we should be able to pull those in and expose them to the user.